### PR TITLE
Update minder repo

### PR DESCRIPTION
**Pull request by riaankleinhans**
Project was moved to a neutral org.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -76,7 +76,7 @@ The following Technical Initiatives have been approved by the TAC. You may learn
...
 | OpenVEX | https://github.com/openvex | [Meeting Notes](https://docs.google.com/document/d/1C-L0JDx5O35TjXb6dcyL6ioc5xWUCkdR5kEbZ1uVQto/edit) | Vulnerability Disclosures WG | [Sandbox](process/project-lifecycle-documents/openvex_for_sandbox_stage.md) |
 | OSV Schema             | https://github.com/ossf/osv-schema               | [Meeting Notes](https://docs.google.com/document/d/1jzqhW9SK9QRA39fQz0RiAkvpRWB0xztt1TAFJEseTlA/edit?usp=sharing) | Vulnerability Disclosures WG   | TBD        |
-| Minder                 | https://github.com/stacklok/minder (TBD)         | New project for inclusion                                                                                         | Security Tooling WG            | [Sandbox](process/project-lifecycle-documents/minder_sandbox_stage.md) |
+| Minder                 | http://github.com/mindersec/minder         | New project for inclusion                                                                                         | Security Tooling WG            | [Sandbox](process/project-lifecycle-documents/minder_sandbox_stage.md) |
```
Comment (lehors) [inline README.md:79]:
```suggestion
| Minder                 | https://github.com/mindersec/minder         | New project for inclusion                                                                                         | Security Tooling WG            | [Sandbox](process/project-lifecycle-documents/minder_sandbox_stage.md) |
```

Comment (riaankleinhans) [reply README.md]: Good catch. Updated. Thank you.

**Review by lehors — APPROVED** [review]
LGTM

**Review by marcelamelara — APPROVED** [review]
LGTM ! Thank you @riaankleinhans

**Review by steiza — APPROVED** [review]

**Pull request merged**